### ISSUE TITLE
Code

### ISSUE THREAD

**Issue by ShinyJJwei** [mubastan/osld#1]
Thank for your sharing the dataset!Will the paper code be available?

**Comment by mubastan** [comment]
Unfortunately, will not be released; but fortunately it is easy to implement and train: (1) train an object detector using the bbox annotations, (2) train an image match model using the image pairs.

**Comment by ShinyJJwei** [comment]
Thank you for your reply, but I have just encountered difficulties in training the matching model. Firstly, I would like to ask whether training with low resolution crop images and high resolution canonical images will affect the final results. In addition, I use proxy anchor loss to train the embedded model and  I enhance the training set to 12 images per class through rotation and changing color. I use resnet50 to extract features, my query  set are the cropped images and gallery  set are  the canonical images, but the recall@1 is only 60%, What should I do to get higher accuracy?

**Comment by mubastan** [comment]
You can use better models than ResNet50, e.g., ConvNeXt, and follow the tips for training the matching model in the paper. Proxy Anchor is one of the best loss functions (should work well, but was not used in the paper); you can experiment with other loss functions too.

Btw, the reported matching accuracies in the paper are average precision (AP), not recall@1.
